opal-core/src/main/java/org/obiba/opal/core/service/VCFSamplesMappingServiceImpl.java

```diff
@@ -70,6 +70,15 @@ public void delete(@NotNull String projectName) throws NoSuchVCFSamplesMappingEx
   }
 
   @Override
+  public void deleteProjectSampleMappings(@NotNull String project) {
+    Iterable<VCFSamplesMapping> list = orientDbService.list(
+        VCFSamplesMapping.class,
+        "select from " + VCFSamplesMapping.class.getSimpleName() + " where tableReference like ?",
+        project + ".%");
+    list.forEach(s -> delete(s.getProjectName()));
```

Comment:
Oh ok good, you delete mapping when it is used by the other projects as well.
And what if the table is deleted all alone?